.github/workflows/main.yml

```diff
@@ -163,27 +163,6 @@ jobs:
           --exclude='/doxygen/xml' --delete --delete-excluded \
           "${DOCS_OUTPUT_DIR}" ${RSYNC_USER}@${RSYNC_SERVER}:${RSYNC_DEST}
 
-  ubuntu-1804-py2:
```

Comment:
I believe this has to remain

Reply:
Ubuntu 18.04 using Python 3 to run SCons is tested in the first build job. This only removes the case where Python 2 (that is, the system Python) is used to run SCons, which is no longer possible with this PR.

Reply:
is there a way to check that the correct error is thrown in a GH action?